app/airq/commands/donate.py

```diff
@@ -0,0 +1,17 @@
+from flask_babel import gettext
+
+from airq.commands.base import MessageResponse
+from airq.commands.base import RegexCommand
+from airq.models.events import EventType
+
+
+class ShowAbout(RegexCommand):
```

Comment:
Can you rename this to `ShowDonate`?

Reply:
You will also need to add it to the list of commands here: https://github.com/airq-dev/hazebot/blob/master/app/airq/commands/__init__.py#L23-L46

And import it here: https://github.com/airq-dev/hazebot/blob/master/app/airq/commands/__init__.py#L6-L7